src/javascript/app/pages/user/account/settings.js

```diff
@@ -6,28 +6,22 @@ const Settings = (() => {
     const onLoad = () => {
         BinarySocket.wait('get_account_status').then(() => {
             const $class_real  = $('.real');
-            const is_jp_client = Client.isJPClient();
 
             if (Client.get('is_virtual')) {
                 $class_real.setVisibility(0);
             } else {
-                $class_real.not((is_jp_client ? '.ja-hide' : '')).setVisibility(1);
+                $class_real.not((Client.isJPClient() ? '.ja-hide' : '')).setVisibility(1);
             }
 
             const status = State.getResponse('get_account_status.status');
             if (!/social_signup/.test(status)) {
                 $('#change_password').setVisibility(1);
             }
 
-            const financial_company = State.getResponse('landing_company.financial_company.shortcode');
-            // Professional Client menu should only be shown to MF and CR accounts.
-            if (!is_jp_client && !/professional_requested|professional/.test(status) &&
-                (Client.isAccountOfType('financial') ||
-                    (/costarica/.test(financial_company) && Client.isAccountOfType('real')))) {
+            // Professional Client menu should only be shown to MF accounts.
+            if ((Client.isAccountOfType('financial')) && !/professional_requested|professional/.test(status) && Client.canRequestProfessional()) {
```

Comment:
we can just use `Client.get('landing_company_shortcode') === 'maltainvest'` instead of `Client.isAccountOfType('financial') && Client.canRequestProfessional()` since it covers both conditions

Reply:
also `/professional/.test(status)` covers both professional and professional_requested regex, if you want to test for the exact values, you can change it to `/^(professional_requested|professional)$/.test(status)`. I know it's from the previous code, but it's good to fix it